test_run.lua

```diff
@@ -102,7 +102,7 @@ local create_cluster_cmd2 = 'start server %s with wait_load=False, wait=False'
 local create_cluster_cmd2_args = create_cluster_cmd2 .. ', args="%s"'
 
 local function create_cluster(self, servers, test_suite, opts)
-    local opts = opts or {}
+    opts = opts or {}
```

Comment:
As already [said](https://lists.tarantool.org/pipermail/tarantool-patches/2020-June/017308.html), I would not forbid redefinitions / hiding variables. Can we place a .luacheckrc file into the repository root to achieve this?

Reply:
My bad, I have pushed changes made before our discussion. New patch contains reverted redefinitions back.